exercises/concept/bird-watcher/.docs/hints.md

```diff
@@ -20,7 +20,8 @@
 
 ## 4. Check if there was a day with no visiting birds
 
-- The `Array` class has a built-in method that returns the first index where the element is found, or -1 if no matching element was found.
+- The `Arrays` class has a built-in method that returns the first index where the element is found, or a negative value if no matching element was found.
+- The array can be iterated over using a `for` loop.
```

Comment:
Let's flip the order since our exemplar is using the `for` loop and we are steering students towards the exemplar.

```suggestion
- The array can be iterated over using a `for` loop.
- The `Arrays` class has a built-in method that returns the first index where the element is found, or a negative value if no matching element was found.
```

We might even consider naming the specific method here (and perhaps even linking to Javadoc).